src/index.ts

```diff
@@ -126,7 +134,7 @@ export class WriteStream extends Writable {
   }
 
   _cleanupSync = (): void => {
-    process.removeListener("exit", this._cleanupSync);
+    processExitProxy.removeListener("exit", this._cleanupSync);
```

Comment:
```suggestion
    processExitProxy.off("exit", this._cleanupSync);
```

https://nodejs.org/dist/latest-v15.x/docs/api/events.html#events_emitter_off_eventname_listener